src/backend/marsha/bbb/serializers.py

```diff
@@ -28,6 +29,8 @@
 class ClassroomSerializer(serializers.ModelSerializer):
     """A serializer to display a Classroom resource."""
 
+    invite_token = serializers.SerializerMethodField(source="get_invite_token")
```

Comment:
Can you add it with the other decalred serializer field ? Also setting `source` seems useless, the `SerializerMethodField` redefine it with `*`